### ISSUE TITLE
Accessibility Feedback for Project Kickoff - Healthcare [Questionnaires]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#11797]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer (VSA) & Trevor (VSP)

## Description

This ticket will house accessibility feedback for theHealthcare [Questionnaires].

### Should

* While the [VSP Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md) does not currently have touchpoints for generative and exploratory research or a "dev intent", for VSA these collaboration opportunities with accessibility **should** happen. Feel free to message Jennifer in Slack, or schedule time on her calendar. 

### Consider

* When planning research — whether it is exploratory, generative, or usability testing — **consider** following the [Inclusive Research Recommendations](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/inclusive-research-recommendations.md#inclusive-research-recommendations), to ensure recruiting is as diverse as possible and that research sessions are accessible.

* **Consider** staying in close touch with the `covid-screener` team and following the #va-how-to-apply-wizards Slack channel for mobile and accessibility enhancements for form patterns. It sounds like these questionnaires are likely to incorporate a number of user touchpoints and range of devices, and optimizing the mobile experience may be beneficial. 

* When designs begin, **consider** a mobile-first approach. This will help clarify content hierarchies, ensure the various mobile use cases are considered from the start, and then smooth the larger viewport design process.

* **Consider** opportunities for progressive enhancement. What information is the most vital and should be available immediately, what can be deferred? What enhancements might be offered for people on wifi or a computer?

## Definition of done

- [X] Review and acknowledge feedback.

- [x] Document decisions made.

- [ ] Accessibility specialist will close ticket after reviewing documented decisions.

cc @1Copenut ready for VSP's review

**Comment by 1Copenut** [comment]
@jenstrickland I concur with the feedback here. I would only add one question. The team mentioned a way for veterans to enter medical data without logging in, and I was intrigued and confused by that. My feedback on that would be to consider building a flow diagram or a written statement outlining how the team would consider this working.

**Comment by abuckley3385** [comment]
@1Copenut - I appologize for the miscommunication i didnt mean for anyone to think we would show FHIR medical information without authentication. The "concept" at this pint would be to fill out minimal inforamtion that is acceptable and valuable without authentication if the Veteran is not able to authorize with the facility to validate they are who they are. to se see thier medical information or login on va.gov to see saved questionnaire forms.

**Comment by abuckley3385** [comment]
@jenstrickland - Im not clear what specifically this ticket is for?  I woud assume assessability feedback is specific to a user story and feature specific which we haven't landed yet.

**Comment by jenstrickland** [comment]
@abuckley3385 

Inclusive design and accessibility are included throughout a project, not just in development. We provide guidance to use as the discovery and design work progresses. We can also identify KPIs to measure the UX quality, if teams are ready to start doing that. As research is created, it should be reviewed by accessibility as noted above. We want to be as inclusive as possible in research activities, and ensure we protect anyone who wouldn't be able to participate from encountering an inaccessible study. Most of the Should and Consider items above are items your designer @arickalewis1 to review and acknowledge.

**Comment by abuckley3385** [comment]
@jenstrickland  I am fully on board getting you pulled in. 

Im still unclear what exactly this issue is to "approve" since is says project kickoff and really just has general design and best practices in the Consider section.
- Is the purpose to review the prototype we mentioned in kickoff that we are doing research on to prioritize our features?  If so, absolutely and I have created a Collab Cycle issue [Usability Testing Prep Questionnaires MVP Priorities #11838](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11838)  (its not filled out yet) to kick off the work with our shared services team and platform.  

If Im understanding correctly then this issue would be an outcome of #11838  and you would be providing accessibility feedback specific to [Research plan for Veteran prioritization of focus areas #11544](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11544) that @arickalewis1  is currently working on this sprint.

**Comment by jenstrickland** [comment]
The Definition of done provides guidance on the steps to review this guidance. In the project kickoff, it's about introducing VSP to your new work initiative. Then, feedback will discuss existing solutions in the problem space, align team member expectations, and collaborate with VSP to identify any areas that may need additional support. That's what the feedback above aims to provide.

**Comment by abuckley3385** [comment]
Should

> generative and exploratory research or a "dev intent", for VSA these collaboration opportunities with accessibility should happen. 

- We scheduled a design/usability testing review before or first round of user testing bringing in the collab cycle team in early to our product. 

Consider

> When planning research — whether it is exploratory, generative, or usability testing — consider following the Inclusive Research Recommendations, to ensure recruiting is as diverse as possible and that research sessions are accessible.

- This was incorporated per comment above from Aricka when she sent to recruiting for participants. 

> Consider staying in close touch with the covid-screener team and following the #va-how-to-apply-wizards 

- We have already met during discovery with both teams to see the patterns used and will follow the slack channel. 

> When designs begin, consider a mobile-first approach. 

- This will occur when we are doing detailed design mocks as it will be expected to have both desktop and mobile view side by side during reviews. 

> Consider opportunities for progressive enhancement. What information is the most vital and should be available immediately, what can be deferred? What enhancements might be offered for people on wifi or a computer?

- yes this is always considered to simplify the experience
